earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -285,90 +284,79 @@ void PacketGenAssetVersionImplD::UpdateChildren(
       prevTime = curTime;
 
       // populate inputs and inputVers & config.insets
```

Comment:
For `levInputs`, `levInputVers`, and 'packetLevelConfig.insets`:
- the calls to `reserve` are no longer necessary since `resize` is being called for each
- I _think_ that the `resize` size for `levInputs` and `levInputVers` should be `mergeIndexes.size() * 2 + 1`. The final `push_back` call after the merge index loop would cause a reallocation and copying of all of the elements otherwise.
- similarly for `packetLevelConfig.insets.resize()`, I think the value should be `mergeIndexes.size() + 1`.